lib/cisco_node_utils/interface.rb

```diff
@@ -933,7 +933,11 @@ def default_switchport_mode
     end
 
     def switchport_trunk_allowed_vlan
-      config_get('interface', 'switchport_trunk_allowed_vlan', name: @name)
+      vlans = config_get('interface', 'switchport_trunk_allowed_vlan',
+                         name: @name)
+      vlans = vlans.join(',') if vlans.is_a?(Array)
+      vlans = Utils.normalize_range_array(vlans, :string) unless vlans == 'none'
```

Comment:
I was assuming `none` was coming back from the yaml in the default case.  I did not realize you could actually configure a value of `none` on the device.  All is good then 👍